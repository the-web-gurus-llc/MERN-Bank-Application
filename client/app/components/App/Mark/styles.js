import styled from 'styled-components';
import { media } from 'utils';

export const StyledMark = styled.img`
  color: black;

  ${media.tablet`
    position: absolute;
    right: 25px;
  `}
`;
