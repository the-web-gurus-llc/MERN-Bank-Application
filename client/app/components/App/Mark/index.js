/**
 *
 * Sidebar
 *
 */

import React from 'react';
import icon from 'images/icon.png';
import { StyledMark } from './styles';

export default function Mark() {
  return <StyledMark width="40" height="auto" src={icon} />;
}
