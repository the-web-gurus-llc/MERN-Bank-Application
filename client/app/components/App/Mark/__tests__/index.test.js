import React from 'react';
import { render } from '@testing-library/react';

import Mark from '../index';

describe('<Mark />', () => {
  it('should render a Mark', () => {
    const { container } = render(<Mark />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
