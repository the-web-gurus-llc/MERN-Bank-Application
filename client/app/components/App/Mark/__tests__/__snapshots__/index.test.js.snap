// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Mark /> should render a Mark 1`] = `
<img
  class="Markstyle__StyledMark-sc-1x4t4ce-0 icqNNZ"
  height="auto"
  src="IMAGE_MOCK"
  width="40"
/>
`;
